src/cases/containers/case-hearings/case-hearings.component.ts

```diff
@@ -53,6 +80,22 @@ export class CaseHearingsComponent implements OnInit, OnDestroy {
     }
   }
 
+  public convertServiceToVM(hearings: CaseHearingModel[]) {
+    const viewModels: CaseHearingViewModel[] = [];
+    hearings.forEach((hearing) => {
+      const viewModel = new CaseHearingViewModel();
+      Object.keys(hearing).forEach(key => viewModel[key] = hearing[key]);
+
+      if (hearing.hearingDaySchedule && hearing.hearingDaySchedule.length) {
+        viewModel.creationDateTime = Math.max.apply(null, hearing.hearingDaySchedule.map(schedule => schedule.hearingStartDateTime ?
+        new Date(schedule.hearingStartDateTime) : new Date(-8640000000000000))) as string;
```

Comment:
what is this new Date(-8640000000000000))) as string for?

Reply:
min date

Reply:
This is a min date I could put it in a constant so it could more readable